contentcuration/contentcuration/tests/test_garbage_collect.py

```diff
@@ -75,6 +75,33 @@ def test_deletes_associated_files(self):
         # there should be no file object in the DB
         assert File.objects.count() == 0
 
+    def test_doesnt_delete_shared_files(self):
```

Comment:
Can you add a docstring explaing what you're testing, and how you're testing it?